model.go

```diff
@@ -18,14 +18,42 @@ type AuthToken struct {
 
 // Record is the lab test results of the user
 type Record struct {
-	ID             int `json:"id,string,omitempty"`
-	User_id        int `json:"userId,string,omitempty"`
-	Age            int `json:"age,string,omitempty"`
-	Height         int `json:"height,string,omitempty"`
-	Weight         int `json:"weight,string,omitempty"`
-	Cholesterol    int `json:"cholesterol,string,omitempty"`
-	Blood_pressure int `json:"bloodPressure,string,omitempty"`
-	Tip_sent       int `json:"tipSent,string,omitempty"`
+	ID                    int  `json:"id,string,omitempty"`
+	User_id               int  `json:"userId,string,omitempty"`
+	Age                   int  `json:"age,string,omitempty"`
+	Height                int  `json:"height,string,omitempty"`
+	Weight                int  `json:"weight,string,omitempty"`
+	Cholesterol           int  `json:"cholesterol,string,omitempty"`
+	Blood_pressure        int  `json:"bloodPressure,string,omitempty"`
```

Comment:
same with this and the ones below. Should be `BloodPressure` and `NumberOfCysts` etc.